src/main/java/com/budgetapp/entities/Category.java

```diff
@@ -0,0 +1,21 @@
+package com.budgetapp.entities;
+
+
+import jakarta.persistence.Entity;
+
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "Category")
```

Comment:
Table names should be in plural form

```suggestion
@Table(name = "Categories")
```